static/src/stylesheets/layout/_header.scss

```diff
@@ -179,3 +147,648 @@ $c-guardian-services-action: #ffffff;
+    // Without this, in Safari/IOS this element appears above the menu
+    z-index: $zindex-overlay;
+    display: none;
+}
+
+.main-menu-container__menu {
+    height: 100%;
+    background-color: $guardian-brand-dark;
+    color: #ffffff;
+    font-family: $f-sans-serif-text;
+    text-transform: lowercase;
+    overflow: auto;
+    -webkit-overflow-scrolling: touch;
+    font-size: 20px;
+    will-change: transform;
+    transition: transform .2s cubic-bezier(.23, 1, .32, 1);
+
+    @include mq($until: mobileMedium) {
+        padding-top: $gs-baseline / 3;
+    }
+
+    @include mq(mobileMedium) {
+        //Larger font/icon size for larger devices
+        font-size: 22px;
+        padding-top: $gs-baseline;
+    }
+
+    .main-menu-container.off-screen & {
+        transform: translateX(-110%);
+        transition: transform .4s cubic-bezier(.23, 1, .32, 1);
+    }
+}
+
+.new-header__nav__button {
+    display: none;
+}
+
+.main-menu-container {
+    transform: translateX(-110%);
+    transition: transform .4s cubic-bezier(.23, 1, .32, 1);
+
+    @include mq($until: mobileMedium) {
+        margin-right: $gutter-small + ($veggie-burger-small / 2);
+    }
+
+    @include mq($from: mobileMedium, $until: mobileLandscape) {
+        margin-right: $gutter-medium + ($veggie-burger-medium / 2);
+    }
+
+    @include mq(mobileLandscape) {
+        margin-right: $gutter-large + ($veggie-burger-medium / 2);
+    }
+}
+
+.new-header__nav__button {
+    &:checked,
+    &[aria-expanded='true']  {
+        & ~ .main-menu-container {
+            transform: translateX(0%);
+            transition: transform .4s cubic-bezier(.23, 1, .32, 1);
+        }
+    }
+    &:checked,
+    &[aria-expanded='true'] {
+        & ~ .main-menu-container__overlay {
+            opacity: 1;
+            transition: opacity .4s cubic-bezier(.23, 1, .32, 1);
+            display: block;
+        }
+    }
+}
+
+.main-navigation {
+    // Unset ul from _lists.scss
+    margin-left: 0;
+    // Hides the line above the first primary item
+    margin-top: -1px;
+    // Unset ul from user agent stylesheet
+    margin-bottom: 0;
+    text-transform: lowercase;
+    // flexbox is being used for reordering the lists on click. Ordering is added by JS
+    // See PR: https://github.com/guardian/frontend/pull/13905
+    display: flex;
+    flex-wrap: wrap;
+    overflow: hidden;
+}
+
+.main-navigation__item {
+    // Override inherited ul from user agent stylesheet
+    list-style: none;
+    color: $news-support-1;
+    width: 100%;
+
+    &:last-child > details[open] {
+        margin-bottom: $gs-baseline;
+    }
+}
+
+//This variable is only used in open menu styling
+$navigation-horizontal-padding: $gs-gutter * 2 + $gs-gutter / 2;
+
+.main-navigation__item__button {
+    position: relative;
+    padding-top: $gs-baseline / 2;
+    padding-left: $navigation-horizontal-padding;
+    padding-bottom: $gs-baseline + ($gs-baseline / 3);
+    cursor: pointer;
+    font-size: 1em;
```

Comment:
Same reason as before. Just making sure it resizes correctly if that parent font-size changes. I think we are okay to remove it though, I imagine we won't be changing the font sizes very much now that the design is finalised for mobile